### PR TITLE
Add Morse Reference

### LINKED ISSUE #60: Morse Code Reference
Inputting characters is nice, but sometimes I just want a table.

### LINKED ISSUE #66: Make Data Entry the primary tab, Reference Secondary
When someone goes to Resistor/Semaphore/Braille/Morse, they should be taken to the encoding entry tool, not the reference.  Change the tab order.

### PR DESCRIPTION
**Pull request by beckbria**
Fixes #60 
Fixes #66 

image

### CODE REVIEW

File: src/Morse/MorseTable.css [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+    
+  .MorseTable th {
+    text-align: center;
+  }
+
+  .MorseTable td {
+    text-align: center;
+    font-size: 40pt;
+    font-weight: bold;
+  }
```
Comment (kfarnung) [inline src/Morse/MorseTable.css]: Weird indents again here

**Comment by kfarnung** [comment]
Does it make sense to use the morse SVGs from the tool?  There's code that turns the ASCII characters into the SVG represenation.

File: src/Morse/MorseTable.tsx [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+      </div>
+    );
+  }
+
+  private renderRows() {
+    const characters = new Character().getPotentialMatches();
+    return characters.map((entry) => (
+      <tr key={entry.display}>
+        <td>{entry.display}</td>
+        <td>{Character.toMorseString(entry.encoding)}</td>
```
Comment (kfarnung) [inline src/Morse/MorseTable.tsx]: I think it would look nicer to use the SVG, but I won't block this change on that.

Comment (beckbria) [reply src/Morse/MorseTable.tsx]: I haven't even realized that we were using SVG here.  I'm torn between looking slightly nicer and being able to copy/paste.  I may use the SVG code for this reference table, but when I write an "Encode" tab, I intend to use plain text for that (unless there's some way to make an SVG have copyable alt-text that I don't know about)

Comment (kfarnung) [reply src/Morse/MorseTable.tsx]: I mean couldn't you just have a "Copy" button that encoded it into a string?  Select and ctrl-c seems like the least friendly way to do that.

Comment (kfarnung) [reply src/Morse/MorseTable.tsx]: I'm also not sure how useful "encode" would be, are there specific scenarios you had in mind?

**Comment by beckbria** [comment]
Writing puzzles, sending coded messages, puzzles that require you to interact with them in an encounter, etc.  I've needed such a thing multiple times over the years and turned to the internet for solutions.
________________________________

**Comment by kfarnung** [comment]
Even though it is supported, replying via email can really make a mess of the comments.

**Comment by beckbria** [comment]
At long last, feedback implemented.  Ready for re-review.

**Comment by kfarnung** [comment]
You should rebase these changes:

```console
$ git fetch --all
$ git rebase origin/master
$ git mergetool
```

I recommend BeyondCompare for resolving merge conflicts, but it's not cheap.

**Pull request merged**